src/pages/docs/chat/getting-started/kotlin.mdx

```diff
@@ -0,0 +1,1040 @@
+    val messages = remember { mutableStateListOf<Message>() }
+
+    DisposableEffect(room) {
+        val subscription = room?.messages?.subscribe { event ->
+            when (event.type) {
+                MessageEventType.Created -> messages.add(0, event.message)
+            }
+        }
+
+        onDispose {
+            subscription?.unsubscribe()
+        }
+    }
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp)
+    ) {
+        Card(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxWidth(),
+        ) {
+            if (messages.isNullOrEmpty()) {
+                Box(
+                    modifier = Modifier.fillMaxSize(),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Text(
+                        text = "No messages yet",
+                        color = Color.Gray
+                    )
+                }
+            } else {
+                LazyColumn(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(8.dp),
+                    reverseLayout = true,
+                ) {
+                    items(messages.size, key = { messages[it].serial }) {
+                        val message = messages[it]
+                        Column(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(vertical = 4.dp)
+                        ) {
+                            Text(
+                                text = "${message.clientId}: ${message.text}",
+                                style = MaterialTheme.typography.bodyMedium
+                            )
+                            Text(
+                                text = SimpleDateFormat("HH:mm:ss", Locale.getDefault())
+                                    .format(Date(message.timestamp)),
+                                style = MaterialTheme.typography.labelSmall,
+                                color = Color.Gray,
+                                modifier = Modifier.align(Alignment.End)
+                            )
+                            HorizontalDivider(modifier = Modifier.padding(top = 4.dp))
+                        }
+                    }
+                }
+            }
+        }
+
+        Spacer(modifier = Modifier.height(8.dp))
+
+        // Message input
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            OutlinedTextField(
+                value = textInput,
+                onValueChange = { textInput = it },
+                modifier = Modifier.weight(1f),
+                placeholder = { Text("Type a message") },
+                maxLines = 3
+            )
+
+            Spacer(modifier = Modifier.width(8.dp))
+
+            Button(
```

Comment:
Something I've noticed here, I have to important _so many_ additional types at this step. Is there some other way we could approach this that this is easier for people, e.g. "here's the imports you need to add"?